modules/amf-registration/amf-registration-web-rest-js/amf-registration-web-rest/src/main/java/com/acme/registration/web/AcmeAccountRegistrationApplication.java

```diff
@@ -129,6 +134,33 @@ public Response getAccount(
 		).build();
 	}
 
+	@GET
+	@Produces(MediaType.APPLICATION_JSON)
+	public Response getAllAccounts(){
+		List<AccountDTO> accountsDTO = new ArrayList<>();
+
+		List<Account> accounts = AccountLocalServiceUtil.getAccounts(-1,-1);
+
+		for (Account account : accounts) {
+			accountsDTO.add(AccountDTO.fromAccount(account));
+		}
+
+		try {
+			String json = _OBJECT_MAPPER.writeValueAsString(accountsDTO);
+
+			return Response.ok(
+					json, MediaType.APPLICATION_JSON
+			).build();
+		}
+		catch (JsonProcessingException jPE) {
```

Comment:
@Odrakir1 I think the source formatter will complain about this. It should be `JsonProcessingException jsonProcessingException.`

Reply:
Hey @mlan0, thank you for this reminder, I've ran the SF, it changed some lines, but it didn't complain about this. Could you please take a look at [this link for reference](https://github.com/liferay/liferay-portal/search?q=ParseException+pe)? It seems to be optional using the whole exception name or its initials. Thank you!